### PR TITLE
removed confusing note

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9072

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/whats-new/ltsc/whats-new-windows-10-2019.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The Windows 10 Enterprise LTSC 2019 release is an important release for LTSC use
 
 ## Microsoft Intune
 
->Microsoft Intune supports Windows 10 Enterprise LTSC 2019 and later.  This includes support for features such as [Windows Autopilot](#windows-autopilot). However, note that Windows Update for Business (WUfB) does not currently support any LTSC releases, therefore you should use WSUS or Configuration Manager for patching.
+Microsoft Intune supports Windows 10 Enterprise LTSC 2019 and later.  This includes support for features such as [Windows Autopilot](#windows-autopilot).
```
Comment (JohanFreelancer9) [inline windows/whats-new/ltsc/whats-new-windows-10-2019.md:39]:
```suggestion
Microsoft Intune supports Windows 10 Enterprise LTSC 2019 and later. This includes support for features such as [Windows Autopilot](#windows-autopilot). 
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - not related to the PR but please remove the extra space.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@greg-lindsay, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by greg-lindsay** [comment]
If the note is confusing it should not be fixed by deleting content. We can clarify or change the wording, but I am not approving a simple deletion.

**Comment by MaratMussabekov** [comment]
Dear @greg-lindsay ,
I believe the note should be removed, as there are ways to update Windows using Intune, which makes the note technically incorrect. Please see the discussion of the issue https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9072 .
Thank you

**Comment by greg-lindsay** [comment]
Hi, the note is about WufB, not Intune. I did not write the note  but accepted it from another contributor who I believe was a Microsoft employee. We need clarification here if there are details being missed, not a simple deletion.

**Comment by MaratMussabekov** [comment]
Dear @greg-lindsay ,
please have a look at https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9231.

Thank you

**Pull request closed without merging**